src/data/machines.js

```diff
@@ -145,175 +145,176 @@ export const ephemeralExternalIpAddressPrice = { // per hour in dollars for all
     n1HourlyGBRamPrice: 0.005458, n1HourlyCpuPrice: 0.04073, preemptibleN1HourlyGBRamPrice: 0.00109, preemptibleN1HourlyCpuPrice: 0.00815,
     t4HourlyPrice: 0.41, p4HourlyPrice: NaN, k80HourlyPrice: NaN, v100HourlyPrice: NaN, p100HourlyPrice: NaN,
     preemptibleT4HourlyPrice: 0.069841, preemptibleP4HourlyPrice: NaN, preemptibleK80HourlyPrice: NaN,
     preemptibleV100HourlyPrice: NaN, preemptibleP100HourlyPrice: NaN
   },
   {
-    name: 'EUROPE-WEST3', monthlyStandardDiskPrice: 0.048, monthlySSDDiskPrice: 0.204,
+    name: 'EUROPE-WEST3', monthlyStandardDiskPrice: 0.048, monthlySSDDiskPrice: 0.204, monthlyBalancedDiskPrice: 0.12,
     n1HourlyGBRamPrice: 0.005458, n1HourlyCpuPrice: 0.04073, preemptibleN1HourlyGBRamPrice: 0.00109, preemptibleN1HourlyCpuPrice: 0.00815,
     t4HourlyPrice: 0.41, p4HourlyPrice: NaN, k80HourlyPrice: NaN, v100HourlyPrice: NaN, p100HourlyPrice: NaN,
     preemptibleT4HourlyPrice: 0.11, preemptibleP4HourlyPrice: NaN, preemptibleK80HourlyPrice: NaN,
     preemptibleV100HourlyPrice: NaN, preemptibleP100HourlyPrice: NaN
   },
   {
-    name: 'EUROPE-WEST4', monthlyStandardDiskPrice: 0.044, monthlySSDDiskPrice: 0.187,
+    name: 'EUROPE-WEST4', monthlyStandardDiskPrice: 0.044, monthlySSDDiskPrice: 0.187, monthlyBalancedDiskPrice: 0.11,
     n1HourlyGBRamPrice: 0.004664, n1HourlyCpuPrice: 0.034802, preemptibleN1HourlyGBRamPrice: 0.000987, preemptibleN1HourlyCpuPrice: 0.007325,
     t4HourlyPrice: 0.35, p4HourlyPrice: 0.65, k80HourlyPrice: NaN, v100HourlyPrice: 2.55, p100HourlyPrice: 1.6,
     preemptibleT4HourlyPrice: 0.11, preemptibleP4HourlyPrice: 0.072, preemptibleK80HourlyPrice: NaN,
     preemptibleV100HourlyPrice: 0.74, preemptibleP100HourlyPrice: 0.43
   },
   {
-    name: 'EUROPE-WEST6', monthlyStandardDiskPrice: 0.052, monthlySSDDiskPrice: 0.221,
+    name: 'EUROPE-WEST6', monthlyStandardDiskPrice: 0.052, monthlySSDDiskPrice: 0.221, monthlyBalancedDiskPrice: 0.13,
     n1HourlyGBRamPrice: 0.005928, n1HourlyCpuPrice: 0.044231, preemptibleN1HourlyGBRamPrice: 0.001254, preemptibleN1HourlyCpuPrice: 0.009309,
     t4HourlyPrice: 0.45, p4HourlyPrice: 0.84, k80HourlyPrice: 0.63, v100HourlyPrice: 3.472, p100HourlyPrice: 2.044,
     preemptibleT4HourlyPrice: 0.11, preemptibleP4HourlyPrice: 0.2808, preemptibleK80HourlyPrice: 0.1755,
     preemptibleV100HourlyPrice: 0.962, preemptibleP100HourlyPrice: 0.559
   },
   {
-    name: 'NORTHAMERICA-NORTHEAST1', monthlyStandardDiskPrice: 0.044, monthlySSDDiskPrice: 0.187,
+    name: 'NORTHAMERICA-NORTHEAST1', monthlyStandardDiskPrice: 0.044, monthlySSDDiskPrice: 0.187, monthlyBalancedDiskPrice: 0.11,
     n1HourlyGBRamPrice: 0.004664, n1HourlyCpuPrice: 0.034802, preemptibleN1HourlyGBRamPrice: 0.000987, preemptibleN1HourlyCpuPrice: 0.007325,
-    t4HourlyPrice: NaN, p4HourlyPrice: 0.65, k80HourlyPrice: NaN, v100HourlyPrice: 0.0, p100HourlyPrice: NaN,
+    t4HourlyPrice: 0.385, p4HourlyPrice: 0.65, k80HourlyPrice: NaN, v100HourlyPrice: 0.0, p100HourlyPrice: NaN,
```

Comment:
Was this an intentional change? It stands out to me as the only change in this file that isn't just adding a `monthlyBalancedDiskPrice`.

Reply:
Good catch - hard to say for intention. 
This value was updated from the parse_pricing_api.py file in dsp-scripts repo. [My edits for the PR are here.](https://github.com/broadinstitute/dsp-scripts/pull/122) It seems like the API got data for t4HourlyPrice instead of NaN.